source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,157 @@
+
+### Option 2: Manually Managed Custom Certificates
+
+If you have a requirement for a custom, dedicated certificate, you can now bring it the Global CDN.
+
+1. [Open a support ticket](/docs/support/#ticket-support) with the certificate details required to generate a **Certificate Signing Request** (CSR). Use as few certificates as possible. Domains from multiple environments and sites can be combined, with up to 100 [**Subject Alternative Names**](https://en.wikipedia.org/wiki/Subject_Alternative_Name){.external} (SANs) per certificate.
+
+
+    <div class="panel panel-drop panel-guide" id="accordion">
+    <div class="panel-heading panel-drop-heading">
+    <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#unique-anchor">
+      <h3 class="info panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-wrench"></span> CSR Details</h3>
+    </a>
+    </div>
+    <div id="unique-anchor" class="collapse" markdown="1" style="padding:10px;">
+    ### CSR Details Required {.info}
+
+    When requesting a CSR file, you must provide Pantheon Support with the following information:
+
+     - Common Name (e.g. `example.com`)
+     - DNS Names (e.g. `example.com`:`www.example.com`:`blog.example.com`:`dev.example.com`)
+     - Organization (e.g. `ACME, Inc.`)
+     - Organizational Unit \[optional\] (e.g. `Engineering`)
+     - Country (e.g. `US`)
+     - Locality (e.g. `San Francisco`)
+     - State / Province (e.g. `California`)
+     - Emails addresses \[optional\] (e.g. `someone@example.com`:`admin@example.com`)
+    </div>
+    </div>
+
+2. Pantheon support will provide you with the CSR file, to pass on to your **Certificate Authority** (CA).
+
+3. Once you have a set of certificates from the CA, send us:
+
+    - The end-client certificate
+    - Any intermediate certficates provided by the CA.<br><br>
+
+    Be sure to send these as separate files, not a "chained cert".
+
+4. [Test Before Going Live](#test-before-going-live) (optional, recommended)
+
+5. Disable Let's Encrypt by adding CAA DNS records TODO: fix link #disable-lets-encrypt-with-caa-records-required
+
+6. Update `A` and `AAAA` records provided by Pantheon support.
+
+
+## Test Before Going Live
+
+Test production domain(s) before updating DNS by overriding DNS on your local computer from your local `hosts` file:
+
+```bash
+23.185.0.X       example.com
+23.185.0.X       www.example.com
+```
+
+In the example `hosts` file above, replace `23.185.0.X` with the IP address provided by Pantheon and `example.com` with your domain name.
```

Comment:
We should probably link to materials for how to use the "hosts" file on common operating systems.